modules/Bio/EnsEMBL/Variation/DBSQL/PhenotypeAdaptor.pm

```diff
@@ -265,6 +265,12 @@ sub get_all_phenotype_class_types {
 sub store{
    my ($self, $pheno) = @_;
 
+   # check if phenotype class_attrib_id is present if class_attrib is
```

Comment:
Is the comment correct? I think the code checks if class_attrib is present if class_attrib_id is not?
Do you need the comment?

Reply:
Thanks for checking. The comment is not clear indeed, have update it.